apps/visualization/filters.py

```diff
@@ -53,15 +63,81 @@ class EpiDataGlobalFilter():
     most_recent: auto
     is_global: bool
     is_twelve_month: bool
+    is_regional_chart: bool
 
     def filter_is_global(self, queryset):
-        if self.is_global == False:
-            return queryset.filter(~Q(region='Global'))
-        return queryset.filter(Q(region='Global'))
+        if self.is_global:
+            return queryset.filter(Q(region='Global'))
+        return queryset.filter(~Q(region='Global'))
+
+    def filter_is_twelve_month(self, queryset):
+        if self.is_twelve_month:
+            return queryset.order_by('-context_date')
```

Comment:
Is this logic enough for 12 month data ?

Reply:
@Rup-Narayan-Rajbanshi  This table holds data per month, client will apply limit =12 offset =0 and get 12 months recode